player/js/module.js

```diff
@@ -107,6 +111,7 @@
 
     bodymovinjs.play = play;
     bodymovinjs.pause = pause;
+    bodymovinjs.setLocationHref = setLocationHref;
```

Comment:
hey @bodymovin, for some reason it never got instantiated with imported bodymovin object. Would you assist in debugging this if you see the same issue?

Thanks